actions/image-build_docs/build_sphinx_docs.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
```

Comment:
```suggestion

# This script encapsulates the process of building the Sphinx documentation
# of the NWChemEx project.
#
# Usage:
#   build_sphinx_docs.sh
#
# Notes:
#   - This script assumes Sphinx is installed via PIP into a virtual environment
#     and the environment is activated.

```

Reply:
I don't think the notes apply here: sphinx is already installed into the image and a virtual environment is not necessary. Can I remove this note?

Reply:
That's fine.

Reply:
The pip installation step has been removed.

Reply:
I think you need that install statement, it's for the documentation dependencies which vary from repo to repo

Reply:
> I think you need that install statement, it's for the documentation dependencies which vary from repo to repo

I see. Is it possible to including everything into the image (the image won't be very big, right?), and so that omit this pip installation step?

Reply:
It's presumably possible. You can try it if you want. The only foreseeable problem is that people will be using the requirements.text file for adding dependencies and that's not going to automatically propagate to the workflow. If you really want to get rid of the bash script, I think you're probably better off looking for a Sphinx action, which presumably would actually take care of the requirements.text.

Reply:
Ok, to move forward quickly, I will keep the pip installation step at this time, but put removing this step in my to-do list. Thanks for your explanation!